Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -547,6 +547,12 @@ open class ChatMessageListVC: _ViewController,
         )
     }
 
+    /// Gets called when mentioned user is tapped.
+    /// - Parameter mentionedUser: The mentioned user that was tapped on.
+    open func messageContentViewDidTapOnMentionedUser(_ mentionedUser: ChatUser?) {
```

Comment:
Do we need an optional `ChatUser` here? We shouldn't call the method if there's no user available.

Reply:
Yes this is wrong, updating 👍 Missed this from the previous implementation

Reply:
Fixed it ✅